test/SerialIOTest.cpp

```diff
@@ -3719,21 +3730,21 @@ TEST_CASE( "git_adios2_sample_test", "[serial][adios2]" )
     REQUIRE( mass.count( RecordComponent::SCALAR ) == 1 );
 
     RecordComponent & mass_scalar = mass[ RecordComponent::SCALAR ];
-    REQUIRE( areEqual( mass_scalar.unitSI(), 6.3356338938136719e-27 ) );
+    REQUIRE( areEqual( mass_scalar.unitSI(), 2.9749182215581054e-28 ) );
     REQUIRE( mass_scalar.getDatatype() == Datatype::DOUBLE );
     REQUIRE( mass_scalar.getDimensionality() == 1 );
-    REQUIRE( mass_scalar.getExtent() == Extent{ 0 } );
+    REQUIRE( mass_scalar.getExtent() == Extent{ 96781 } );
```

Comment:
You could also read "mass" and "charge" with `loadChunk` to make sure constant records are read well :)

Reply:
(A bit more as a hack, you can also compare the `value` attribute to it.)

Reply:
added a commit :)